drivers/misc/cldatadrv/r_taurus_bridge.h

```diff
@@ -0,0 +1,140 @@
+
+#ifndef R_TAURUS_BRIDGE_H
```

Comment:
Looks like this is a copy of drivers/gpu/drm/rcar-rvgc/r_taurus_bridge.h

We don't need to copy definition. Move that file into some common place and include it for all TAURUS drivers

Reply:
Why are you adding this file and then removing it in the next patch?